app/views/staff/activity_forms/country.html.haml

```diff
@@ -1,2 +1,2 @@
 = render layout: "wrapper" do |f|
-  = f.govuk_collection_select :recipient_region, yaml_to_objects("activity", "recipient_region"), :code, :name
+  = f.govuk_collection_select :recipient_region, region_select_options, :code, :name, options: { selected: f.object.recipient_region || region_select_options.first.code }
```

Comment:
Ensuring we have the HTML attribute `option selected=selected` on our default option took this change which is worth sharing as we'll need to do it elsewhere.